components/builder-api/src/server/error.rs

```diff
@@ -193,7 +193,9 @@ impl Into<HttpResponse> for Error {
             Error::ParseIntError(_) => HttpResponse::new(StatusCode::BAD_REQUEST),
             Error::Protocol(_) => HttpResponse::new(StatusCode::INTERNAL_SERVER_ERROR),
             Error::BuilderCore(ref e) => HttpResponse::new(bldr_core_err_to_http(e)),
-            Error::DieselError(ref e) => HttpResponse::new(diesel_err_to_http(e)),
+            Error::DieselError(ref e) => {
+                HttpResponse::with_body(diesel_err_to_http(e), e.description().to_string())
```

Comment:
I'm not sure about exposing the description in the error - the changes we've made so far explicitly don't put error text in the body any longer.

Reply:
Yes we can add debug lines